catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -554,11 +555,14 @@ public Response getDocument(@Encoded @PathParam("sourceid") String encodedSource
                 LOGGER.warn(exceptionMessage, e);
                 throw new ServerErrorException(exceptionMessage, Status.BAD_REQUEST);
                 // The catalog framework will throw this if any of the transformers blow up. We need to
```

Comment:
With the addition of the `DataUsageLimitException`, this comment appears out of place. Is this comment intended for the `catch (RuntimeException | UnsupportedEncodingException e) `?